riff-raff.yaml

```diff
@@ -2,6 +2,17 @@ regions: [eu-west-1]
 stacks: [frontend]
 
 deployments:
+  front-pressed-lambda-cloudformation:
+    type: cloud-formation
+    app: front-pressed-lambda
+    parameters:
+      templatePath: frontend.yml
+      # The next few lines make sure the name used is the same as the existing
+      # stack, and that this is updated rather than a new one created.
+      createStackIfAbsent: false
+      cloudFormationStackName: front-pressed-lambda
+      cloudFormationStackByTags: false
```

Comment:
Is `cloudFormationStackByTags` not the preferred approach now?

Reply:
Yes, this is [the default](https://riffraff.gutools.co.uk/docs/magenta-lib/types#cloudformation), so unless there is a reason that we must use name based discovery then I'd use tags. You might need to add the correct tags manually if they are not already present.

Setting `createStackIfAbsent` to `false` is still a good idea because this means that we won't accidentally create a copy of this if the tag-based discovery fails.

Reply:
I think using tag-based discovery would also be more consistent with the other Riff-Raff project deployed via this repo:

https://github.com/guardian/front-pressed-lambda/blob/a07bf73bee6a469cb24ec398933b92ed6bd6c15e/.github/workflows/ci.yml#L77-L83

Reply:
Possibly, but it wasn't finding the existing stack and trying to create a new one instead 🤔. I can see it does now have the correct tags so maybe they weren't there before but RiffRaff has now added them? I'll have another go now.

Reply:
Yeah the tags aren't there in PROD, which has never been deployed with RiffRaff as far as I can tell. But they are there in CODE, so RiffRaff must have added them during deployments testing this PR.

[b0b1a5f](https://github.com/guardian/front-pressed-lambda/pull/45/commits/b0b1a5f2243a0c28fe7b0514ad4bfaffd1f6277c) now works in CODE. I'll add the tags manually to the PROD stack so we can deploy using tag lookup after merge.